### DISCUSSION TITLE
how-to-supercharge-terminal/

### DISCUSSION THREAD

**Discussion by giscus** [Dhravya/blog#9] [General]
# how-to-supercharge-terminal/

How I customized my terminal  to look really good using Starship, Powershell and Nerd Fonts

https://blog.dhravya.dev/how-to-supercharge-terminal/

**Comment by nexxeln** [comment]
sexy

**Reply by Dhravya** [reply]
wow first comment 🥺

**Comment by SpaceCypher** [comment]
epic

**Reply by Dhravya** [reply]
Thanks!
